.github/workflows/labeler.yml

```diff
@@ -2,16 +2,16 @@ name: "Pull Request Labeler"
 on:
   - pull_request_target
 
-permissions:  # added using https://github.com/step-security/secure-workflows
+permissions:
   contents: read
 
 jobs:
   triage:
     permissions:
       contents: read
       pull-requests: write
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Why are we using a specific version here and not just keeping at at latest? Do we risk forgetting to update this or will it be automatically updated with dependabot?

Reply:
We prefer to use specific versions to avoid ambiguity. We have `ubuntu-22.04` in all the other workflows and this change is to align all of them.

Dependabot won't update this, we'll have to do it manually but (historically) they only use Ubuntu LTS versions so the update will be every couple of years